testsuite/tests/contrib/lkt_semantic/decl_block_correctness/test.out

```diff
@@ -1,25 +1,5 @@
 Resolving test.lkt
 ==================
-test.lkt:5:4: error: value declaration forbidden in trait declaration
```

Comment:
I realize now that we have error messages in a disturbing order. Would it be possible to enhance either a property or lkt_toolbox to sort error messages? Test updates would be much easier to review, and I think it would be user friendlier for users too. WDYT?

Reply:
I'm not against it at first sight, but won't do this as part of this PR, since this was already the case before, and I need to think on it a bit to see if it doesn't have any hidden drawbacks